test/ItspServices.pServer.Test/AuthorizedEndpointTest.cs

```diff
@@ -182,7 +127,7 @@ public async Task ProtectedDataGetRootFolder()
                     parentId = default(int?),
                     name = "root",
                     protectedDataIds = default(int[]),
-                    subfolderIds = new[] { 1 }
+                    subfolderIds = default(int[])
```

Comment:
I wouldn't remove that check that the subfolders also sent via the API.
You don't need to do anything here, I will make a commit to solve this.